proto/minder/v1/minder.proto

```diff
@@ -1887,27 +1887,64 @@ message ListEvaluationResultsResponse {
 message RestType {
     message Fallback {
         int32 http_code = 1 [(buf.validate.field).int32 = { gte: 100, lte: 599 }];
-        string body = 2;
+        string body = 2 [
+            (buf.validate.field).string = {
+                pattern: "^\\s*\\{.*\\}\\s*$",
+                max_len: 1000,
+            },
+            (buf.validate.field).ignore = IGNORE_IF_DEFAULT_VALUE
+        ];
     }
 
     // endpoint is the endpoint to fetch data from.
     // This can be a URL or the path on the API.bool
     // This is a required field and must be set.
     // This is also evaluated via a template which allows
     // us dynamically fill in the values.
-    string endpoint = 1;
+    string endpoint = 1 [
+        (buf.validate.field).string = {
+            pattern: "^(https:\\/\\/[a-zA-Z0-9.-]+)?(\\/[a-zA-Z0-9._-]+)*(\\/\\{\\{[^}]+\\}\\})*(\\/)?$",
```

Comment:
Is this a "URI with Go template values embedded"?  Again, CEL might be an option, or combining prefix with some other expressions.

Reply:
Yes, I updated it to `"^(https?:\\/\\/)?[a-zA-Z0-9._-]+(\\/[a-zA-Z0-9._{}\\/-]*)?$"` which should allow for:
* "https://api.something.com"
* "https://api.something.com/path/to/resource"
* "/repos/{{.Entity.Owner}}/{{.Entity.Name}}/branches/main"
* "{{ $branch_param := index .Params \"branch\" }}/repos/{{.Entity.Owner}}/{{.Entity.Name}}"